plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -1329,7 +1329,25 @@ void GSRendererOGL::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sour
 		vs_cb.TextureScale = vs_cb.TextureScale.xyxy(point_size);
 	}
 #endif
-	m_gs_sel.sprite = m_vt.m_primclass == GS_SPRITE_CLASS;
+	m_gs_sel.sprite = m_vt.m_primclass == GS_SPRITE_CLASS || m_vt.m_primclass == GS_LINE_CLASS;
+	if ((m_upscale_multiplier > 1) && (m_vt.m_primclass == GS_LINE_CLASS) && GLLoader::found_geometry_shader) {
```

Comment:
``(m_upscale_multiplier > 1)`` means you have excluded custom resolutions, is that intended ?